### DISCUSSION TITLE
Notification not showing in standalone

### DISCUSSION THREAD

**Discussion by vlogvinit** [primefaces/community#62] [PrimeNG] [answered]
Hello.
I'm using the notification component in my Angular 16 standalone web app, but it does not pop up, there is no error message.
Note this is a standalone Angular app so there are no ngModules at all.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

I assume that you're referring toast or messages component by "notification". Could you please share a Stackblitz reproducer by forking this [example](https://stackblitz.com/edit/8n4zfy?file=src%2Fapp%2Fdemo%2Ftoast-basic-demo.ts) so we can investigate and identify the issue clearly?

**Reply by vlogvinit** [reply]
Sorry. I used service injection. Now I'm using MessageService directly and everything works fine.

**Reply by cetincakiroglu** [reply]
Happy to hear that
